cc_dv_bm3024.Rmd

```diff
@@ -0,0 +1,190 @@
+# Plotly R graphing
+
+Brian Mao
+## Introduction
+
+Plotly's R graphing library makes interactive, publication-quality graphs. in this into to Plotly I will show some examples of how to make line plots, scatter plots, area charts, bar charts, box plots, histograms, heatmaps, and 3D scatter plot charts. there are a lots more advanced function in the Plotly. you may explore it later on yourself 
+
+```{r setup, include=FALSE}
```

Comment:
Please follow 2.2.6 (https://jtr13.github.io/cc21/github-submission-instructions.html) and remove 'setup'